corehq/apps/style/static/preview_app/less/preview_app/form.less

```diff
@@ -1,7 +1,8 @@
 .form-container {
-  padding-top: 35px;
   background-color: white;
-  min-height: 403px;
+  min-height: -webkit-calc(~"100vh - " (@webforms-nav-height + @webforms-breadcrumb-height));
+  min-height: -moz-calc(~"100vh - " (@webforms-nav-height + @webforms-breadcrumb-height));
+  min-height: calc(~"100vh - " (@webforms-nav-height + @webforms-breadcrumb-height));
```

Comment:
@biyeun added these calcs which give you this:

<img width="305" alt="screen shot 2017-01-03 at 3 48 23 pm" src="https://cloud.githubusercontent.com/assets/918514/21610470/e4783bfa-d1d0-11e6-8544-dad37d383d37.png">